operations/workspace/deployment/pkg/step/installgitpod.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package step
+
+import (
+	"strings"
+
+	"github.com/gitpod-io/gitpod/ws-deployment/pkg/common"
+	"github.com/gitpod-io/gitpod/ws-deployment/pkg/runner"
+)
+
+const (
+	// DefaultGitpodInstallationScript is the path to script that must be invoked
+	// from its parent dir to install gitpod
+	DefaultGitpodInstallationScript = "dev/build-ws-cluster/install-gitpod.sh"
+)
+
+func InstallGitpod(context *common.ProjectContext, gitpodContext *common.GitpodContext, cluster *common.WorkspaceCluster) error {
+	err := runner.ShellRunWithDefaultConfig(DefaultGitpodInstallationScript, []string{"-g", context.Id, "-l", cluster.Region, "-n", cluster.Name, "-a", getValuesFilesArgString(gitpodContext)})
+	return err
+}
+
+func getValuesFilesArgString(gitpodContext *common.GitpodContext) string {
```

Comment:
Does this work? Shouldn't this be something like 
```suggestion
func getValuesFilesArgString(gitpodContext *common.GitpodContext) []string {
```

Reply:
The string generated by this function looks like:
`-f v1.yaml -f v2.yaml -f v3.yaml`

This is passed as a single argument to the installation script like: -a `-f v1.yaml -f v2.yaml -f v3.yaml`

Then the script will use it while running the helm command:

helm install `-f v1.yaml -f v2.yaml -f v3.yaml`

Reply:
I am wondering if we should pull helm command into our code and let the script only generate the value file.

Reply:
or better yet, generate the value file in Go directly rather than relying on a script